lib/dialects/mysql/connection-manager.js

```diff
@@ -82,6 +81,11 @@ class ConnectionManager extends AbstractConnectionManager {
       supportBigNumbers: true
     };
 
+    // -FOUND_ROWS used by default but can be suppressed
+    if (!config.dialectOptions || config.dialectOptions.foundRows !== false) {
```

Comment:
@sushantdhiman `-FOUND_ROWS` is the default behavior so there's never a need to pass it in this way. What I am trying to introduce is a way to suppress that flag.

If I did it your way the usecase would be for people to specify `dialectOptions.flags = ''`for the default flag to be suppressed. Both ways sound goo to me.

@janmeier we talked about the name of this config property in the past. WDYT about `config.dialectOptions.flags=""` vs `config.dialectOptions.foundRows = false`?